packages/api/src/command/medical/cq-directory/search-cq-directory.ts

```diff
@@ -0,0 +1,106 @@
+import { Coordinates } from "@metriport/core/external/aws/location";
+import convert from "convert-units";
+import { Sequelize } from "sequelize";
+import { geocodeAddresses } from "../../../external/aws/address";
+import { CQDirectoryEntryModel } from "../../../models/medical/cq-directory";
+import { getPatientOrFail } from "../patient/get-patient";
+
+export const DEFAULT_RADIUS_IN_MILES = 50;
+
+export type CQOrgBasicDetails = {
+  name: string | undefined;
+  id: string;
+  lon: number | undefined;
+  lat: number | undefined;
+  urlXCPD: string;
+  urlDQ: string | undefined;
+  urlDR: string | undefined;
+};
+
+/**
+ * Searches the Carequality Directory for organizations within a specified radius of a patient's addresses.
+ * @param cxId The ID of the customer organization.
+ * @param patientId The ID of the patient.
+ * @param radiusInMiles Optional, the radius in miles within which to search for organizations. Defaults to 50 miles.
+ *
+ * @returns Returns the details of organizations within the specified radius of the patient's address.
+ */
+export const searchNearbyCQOrganizations = async ({
+  cxId,
+  patientId,
+  radiusInMiles = DEFAULT_RADIUS_IN_MILES,
+}: {
+  cxId: string;
+  patientId: string;
+  radiusInMiles?: number;
+}) => {
+  const radiusInMeters = convert(radiusInMiles).from("mi").to("m");
+
+  const patient = await getPatientOrFail({ id: patientId, cxId });
+  const coordinates = await geocodeAddresses(patient.data.address);
+
+  const orgs = await searchCQDirectoriesByRadius({
+    coordinates,
+    radiusInMeters,
+  });
+
+  return pickBasicOrganizationAttributes(orgs);
+};
+
+/**
+ * Searches the Carequality Directory for organizations within a specified radius around geographic coordinates.
+ *
+ * @param coordinates The latitude and longitude around which to search for organizations.
+ * @param radiusInMeters The radius in meters within which to search for organizations.
+ * @returns Returns organizations within the specified radius of the patient's address.
+ */
+export const searchCQDirectoriesByRadius = async ({
+  coordinates,
+  radiusInMeters,
+}: {
+  coordinates: Coordinates[];
+  radiusInMeters: number;
+}): Promise<CQDirectoryEntryModel[]> => {
+  const orgs = [];
```

Comment:
give a type for `orgs`. The current code likely means `orgs: any = []`, which means we can add anything on line 84 without the compiler having our backs